atm.js

```diff
@@ -1,3 +1,28 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+$("body").css("background-color", "red");
+
```

Comment:
hey i noticed you marked the assignment's completeness as a 5 but just have a couple lines of code here; do you have code locally that you need to push up?